models/question.ts

```diff
@@ -0,0 +1,41 @@
+import * as uuid from 'uuid';
+
+export enum QuestionType {
+  TEXT = 'text',
+}
+
+export type QuestionData = TextQuestionData;
```

Comment:
Is the idea for assigning this to a type here is so we can have a union of the different question types assigned to this?